Makefile

```diff
@@ -1006,6 +1006,7 @@ IMAGEREADER_OBJ := $(patsubst %.cpp, $(OBJDIR)/%.o, $(IMAGEREADER_SRC))
 IMAGEREADER:=$(LIBDIR)/Cntk.Deserializers.Image-$(CNTK_COMPONENT_VERSION).so
 ALL_LIBS += $(IMAGEREADER)
 PYTHON_LIBS += $(IMAGEREADER)
+JAVA_LIBS += $(IMAGEREADER_LIBS)
```

Comment:
You may consider adding same in ImageWriter section as well but that is optional. ImageWriter is required only to use TensorBoard's imageWrite feature only.

Reply:
Sounds good thank you for the fast review!

Reply:
Done